repair/repair.hh

```diff
@@ -71,6 +71,11 @@ struct node_ops_info {
     utils::UUID ops_uuid;
     shared_ptr<abort_source> as;
     std::list<gms::inet_address> ignore_nodes;
+
+    node_ops_info(utils::UUID ops_uuid_, shared_ptr<abort_source> as_, std::list<gms::inet_address>&& ignore_nodes_) noexcept;
+    node_ops_info(const node_ops_info&) = delete;
+    node_ops_info(node_ops_info&&) = delete;
```

Comment:
Does this also delete assignment ops? I don't recall the rules.

Reply:
yes, it does.